code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -411,6 +411,7 @@
 		icon_state = icon_dead
 		stat = DEAD
 		density = 0
+		lying = 1
```

Comment:
I hope this doesn't cause issues, this is needed so animals with blood leave a blood trail when dead.